data/json/mutations.json

```diff
@@ -612,7 +612,7 @@
     "points": 2,
     "description": "You have taken some self-defense classes at a nearby gym.  You start with your choice of Capoeira, Krav Maga, Muay Thai, Ninjutsu, or Zui Quan.",
```

Comment:
```suggestion
    "description": "You have taken some self-defense classes at a nearby gym.  You start with your choice of Capoeira, Krav Maga, Muay Thai, Ninjutsu, Zui Quan, or Wing Chun.",
```

Reply:
thx

Reply:
The styles are in alphabetical order, so Zui Quan should be the last.